master-firmware/protobuf/computer_vision.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+import "nanopb.proto";
+
+message ComputerVisionResult {
```

Comment:
`std::optional`?

Reply:
I am not sure what you suggest here. This is a proto file, not c++.

Reply:
Someday we'll get optional in protobuf? A man can dream